test/pytest/test_example_stateful_http.py

```diff
@@ -126,7 +126,7 @@ def test_stateful_mar(mar_file_path, model_store):
             args=(
                 model_name,
                 "seq_1",
-                "1 3 6",
+                "1 3 6 10 15",
```

Comment:
Longer sequence is good, but the sequences should be different to make sure we have no cross talking between them. Also why would we expect 1 3 6 for seq_0 if we send 5 requests?